diff -Naur makemkv-oss-1.17.3.old/libffabi/src/ffabi.c makemkv-oss-1.17.3/libffabi/src/ffabi.c
--- makemkv-oss-1.17.3.old/libffabi/src/ffabi.c	2023-04-11 12:43:16.779074232 +0200
+++ makemkv-oss-1.17.3/libffabi/src/ffabi.c	2023-04-11 12:43:38.580475108 +0200
@@ -478,7 +478,7 @@
 #else
     ctx->frame = av_frame_alloc();
     if (av_sample_fmt_is_planar(ctx->avctx->sample_fmt)) {
-        ctx->frame_extended_data = av_mallocz_array(ctx->avctx->channels,
+        ctx->frame_extended_data = av_calloc(ctx->avctx->channels,
             sizeof(*ctx->frame_extended_data));
         if (!ctx->frame_extended_data) {
             ffm_audio_encode_close(ctx);
