diff -Naur makemkv-oss-1.9.4.old/Makefile.in makemkv-oss-1.9.4/Makefile.in
--- makemkv-oss-1.9.4.old/Makefile.in	2015-06-19 09:07:28.119677023 +0200
+++ makemkv-oss-1.9.4/Makefile.in	2015-06-19 09:08:06.705755779 +0200
@@ -46,16 +46,13 @@
 	$(INSTALL) -D -m 644 makemkvgui/share/icons/128x128/makemkv.png $(DESTDIR)$(DATADIR)/icons/hicolor/128x128/apps/makemkv.png
 endif
 
-out/%: out/%.full
-	$(OBJCOPY) --strip-all --strip-debug --strip-unneeded --discard-all $< $@ 
-
-out/libdriveio.so.0.full:
+out/libdriveio.so.0:
 	mkdir -p out
 	$(GCC) $(CFLAGS) -D_REENTRANT -shared -Wl,-z,defs -o$@ $(LIBDRIVEIO_INC) $(LIBDRIVEIO_SRC) \
 	-fPIC -Xlinker -dy -Xlinker --version-script=libdriveio/src/libdriveio.vers \
 	-Xlinker -soname=libdriveio.so.0 -lc -lstdc++
 
-out/libmakemkv.so.1.full: tmp/gen_buildinfo.h
+out/libmakemkv.so.1: tmp/gen_buildinfo.h
 	mkdir -p out
 	$(GCC) $(CFLAGS) -D_REENTRANT -shared -Wl,-z,defs -o$@ $(LIBEBML_INC) $(LIBEBML_DEF) $(LIBMATROSKA_INC) \
 	$(LIBMAKEMKV_INC) $(SSTRING_INC) $(MAKEMKVGUI_INC) $(LIBABI_INC) $(LIBFFABI_INC) $(LIBDCADEC_DEF) \
@@ -65,14 +62,14 @@
 	-fPIC -Xlinker -dy -Xlinker --version-script=libmakemkv/src/libmakemkv.vers \
 	-Xlinker -soname=libmakemkv.so.1 -lc -lstdc++ -lcrypto -lz -lexpat $(FFMPEG_LIBS) -lm -lrt
 
-out/libmmbd.so.0.full:
+out/libmmbd.so.0:
 	mkdir -p out
 	$(GCC) $(CFLAGS) -D_REENTRANT -shared -Wl,-z,defs -o$@ $(MAKEMKVGUI_INC) $(LIBMMBD_INC) \
 	$(LIBMAKEMKV_INC) $(SSTRING_INC) $(LIBABI_INC) $(LIBMMBD_SRC) $(LIBMMBD_SRC_LINUX) $(SSTRING_SRC) \
 	-fPIC -Xlinker -dy -Xlinker --version-script=libmmbd/src/libmmbd.vers \
 	-Xlinker -soname=libmmbd.so.0 -lc -lstdc++ -lrt -lpthread -lcrypto
 
-out/makemkv.full: $(MAKEMKVGUI_GEN) $(MAKEMKVGUI_SRC_LINUX) tmp/gen_buildinfo.h
+out/makemkv: $(MAKEMKVGUI_GEN) $(MAKEMKVGUI_SRC_LINUX) tmp/gen_buildinfo.h
 	mkdir -p out
 	$(GCC) $(CFLAGS) -o$@ $(MAKEMKVGUI_INC) $(LIBMAKEMKV_INC) $(SSTRING_INC) $(LIBDRIVEIO_INC) $(LIBABI_INC) \
 	$(MAKEMKVGUI_SRC) $(MAKEMKVGUI_SRC_LINUX) $(MAKEMKVGUI_GEN) $(SSTRING_SRC) $(LIBDRIVEIO_SRC_PUB) \
